app/models/organization.rb

```diff
@@ -15,11 +11,7 @@ def with_transfer
   has_many :inquiries
   has_many :documents, as: :documentable
 
-  scope :matching, ->(str) {
-    where(Organization.arel_table[:name].matches("%#{str}%"))
-  }
-
-  validates :name, uniqueness: true
+  validates :name, presence: true, uniqueness: true
```

Comment:
let's make sure we also have `null: false` and a unique index at db level 🙏

Reply:
agree @sauloperez I'll review the schema and if in case this is missing, I'll create a migration to fix that

Reply:
@markets can you please check this, thanks!!! (no rush of course :smile: )

Reply:
Oh, I see you did it already here https://github.com/coopdevs/timeoverflow/pull/355/commits/b9d9b9636c806d48aad7e75cd353160572750689 right?